vendor/engines/secure_rooms/db/migrate/20170220203130_add_indala_number_to_user.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndalaNumberToUser < ActiveRecord::Migration
+  def change
+    add_column :users, :indala_number, :string
```

Comment:
One thought I had is around the name of the database field. Indala seems very specific to NU (though, it could be more universal than I expect). What would you think of naming it something like `scan_card_number` or `card_number`. We would then override the locale/human_attribute_name in NU to "Indala Number". And on that note, even if we leave it as is, we should probably set the human_attribute_name to capitalize "Number".